### PR TITLE
Windows refactor

### PR DESCRIPTION
**Pull request by thxCode**
- Don't need `RKEK8sWindowsSystemImage` anymore, Windows keeps its images as the same as `RKEK8sSystemImage` via Docker manifest.
- Add `PodInfraWindowsContainer` filed into `RKESystemImages` to indicate the Windows pause container image name.

**Issue:**
rancher/rancher#16460

### CODE REVIEW

File: apis/management.cattle.io/v3/rke_types.go [hunk 1]
```diff
@@ -93,6 +93,8 @@ type RKESystemImages struct {
 	// Alpine image
 	Alpine string `yaml:"alpine" json:"alpine,omitempty"`
 	// rke-nginx-proxy image
+	// related windows note: have a manifest support for rke-tools to compatible with windows, references:
+	// - https://github.com/rancher/rke-tools/pull/58
```
Comment (orangedeng) [inline apis/management.cattle.io/v3/rke_types.go]: why adding node here?

Comment (thxCode) [reply apis/management.cattle.io/v3/rke_types.go]: Removed

**Comment by orangedeng** [comment]
LGTM

**Comment by orangedeng** [comment]
LGTM

**Pull request merged**